packages/webpack/lib/configs/node.js

```diff
@@ -50,11 +50,11 @@ module.exports = function getConfig(config, name) {
 
   const fileLoaderConfig = {
     exclude: [/\.(?:m?js|html|json)$/],
-    loader: require.resolve('file-loader'),
-    options: {
-      name: getAssetPath('[name]-[hash:16].[ext]'),
-      esModule: false,
-      emitFile: false,
+    type: 'asset/resource',
+    generator: {
+      filename: getAssetPath('[name]-[contenthash:16][ext]'),
+      // TODO: comment in next line as soon as webpack@>5.24.4 supports prefetching
```

Comment:
What has the emitting of assets to do with prefetching?

Reply:
Support for the `emit`-option has been introduced with `webpack@5.25`. So, as soon as webpack supports the prefetching appropriately, we can update to the latest version and can then also comment in that option.